Fix:
./ssh.c:137:2: error: implicit declaration of function 'MD5' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- ssh.c.orig	2001-03-15 02:33:04.000000000 -0600
+++ ssh.c	2021-07-29 12:50:52.000000000 -0500
@@ -15,6 +15,7 @@
 #include <arpa/nameser.h>
 #include <openssl/ssl.h>
 #include <openssl/err.h>
+#include <openssl/md5.h>
 #include <openssl/rand.h>
 
 #include <err.h>
